build/toolchain/android/BUILD.gn

```diff
@@ -60,12 +60,13 @@ template("android_gcc_toolchain") {
 
       cc = compiler_prefix + prefix + "/clang"
       cxx = compiler_prefix + prefix + "/clang++"
+      ar = prefix + "/llvm-ar"
```

Comment:
@jason-simmons - I branched ar here so that if someone isn't using clang they fall back to the NDK ar.